packaging/machine-learning-api.spec

```diff
@@ -12,7 +12,7 @@
 %define		tensorflow_lite_support	1
 %define		tensorflow2_lite_support 1
 %define		tensorflow2_gpu_delegate_support 1
-%define		nnfw_support 1
+%define		nnfw_support 0
```

Comment:
Can you avoid error simply by removing:

https://github.com/nnstreamer/api/blob/80f62746c1d37cf0ca0d9f8d6c67a44bbfb38c20/packaging/machine-learning-api.spec#L141-L142

Reply:
Thanks, Updated.
I checked that OneRT also removed armcl backend onTizen.